2420/2/options_file.rb

```diff
@@ -0,0 +1,61 @@
+require 'optparse'
+require './versus.rb'
+require 'terminal-table'
+
+options = { name: '', top_bad_words: 0, top_words: 30 }
+OptionParser.new do |opts|
+  opts.banner = 'Usage: example.rb [options]'
+
+  opts.on('--top-bad-words [NUMBER]', Integer, 'Максимальное количество самых нецензурных участников') do |number|
+    options[:top_bad_words] = number
+  end
+  opts.on('--name [NAME]', String, 'Имя участника') do |name|
+    options[:name] = name
+  end
+  opts.on('--top-words [NUMBER]', Integer, 'Любимые слова человека') do |number|
+    options[:top_words] = number || 30
+  end
+end.parse!
+
+if options[:top_bad_words] > 0
```

Comment:
move this to a new class, TopBadWordsCounter

it should look like

`TopBadWordsCounter.new(NAMES).print if options[:top_bad_words] > 0`

Reply:
Don't COPYPASTE this code to 1 method in new class!! Separate this to different methods, where only 1 will be public!